charts/aws-vpc-cni/values.yaml

```diff
@@ -49,7 +49,7 @@ env:
 
 # this flag enables you to use the match label that was present in the original daemonset deployed by EKS
 # You can then annotate and label the original aws-node resources and 'adopt' them into a helm release
-originalMatchLabels: false
+originalMatchLabels: true 
```

Comment:
this should be left as false since managed-cni is using false. which renders the k8s standard labels :D

Reply:
Updated the PR. To override in the helm install command is taken care in this PR - https://github.com/aws/amazon-vpc-cni-k8s/pull/1541